app/models/event.rb

```diff
@@ -62,8 +62,40 @@ def self.events_to_json
     timeline.to_json
   end
 
+  def self.events_to_calendar(format = nil, filename)
```

Comment:
Apparently you don't need the filename as a parameter here

Reply:
Thanks @Levia I will fix that!